index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" property="og:title" content="width=device-width">
+    <title>Website</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  
+  <body>
+  <main id="main-doc">
```

Comment:
please note that the `main` section should be after `header` tag and before `footer` tag

Reply:
* Kindly explain why you request a change - it might be obvious for you but not for someone else

Reply:
The `main` tag should wrap the whole content between `header` and `footer ` , kindly change the position of the opening and  the closing tag of main, by this it will be also easy to align elements inside it because all of them centred and have the same margin and padding on the left and right side.